qiskit_ibm_experiment/service/ibm_experiment_service.py

```diff
@@ -1742,6 +1742,31 @@ def file_download(
         )
         return file_data
 
+    def file_delete(self, experiment_id: str, file_pathname: str):
+        """Deletes a data file from the DB
+
+        Note:
```

Comment:
I don't think this note will render correctly, you'll want to use `.. note::` (consult the [sphinx rst primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)). You can build the docs locally with `tox -edocs` and preview them.

Reply:
It works

Reply:
How did you test this? I actually don't see `file_delete()` in the built docs from the [latest run](https://github.com/Qiskit-Extensions/qiskit-ibm-experiment/actions/runs/8735063112?pr=101):
![image](https://github.com/Qiskit-Extensions/qiskit-ibm-experiment/assets/3870315/caf8a65e-eefd-46b1-88c3-4b0a3d59346d)

Reply:
The local `qiskit_ibm_experiment` module is not accessible to sphinx, so it uses the existing pip module, which doesn't have the `file_delete` method.
When I add the path of the local `qiskit_ibm_experiment` to `PYTHONPATH `, sphinx builds the documentations for the new methods.